lib/routes/denonbu/news.ts

```diff
@@ -0,0 +1,200 @@
+| akiba         | 外神田文芸高校                                   |
+| harajuku      | 神宮前参道學園                                   |
+| azabu         | 港白金女学院                                     |
+| shibuya       | 帝音国際学院                                     |
+| kabuki        | 真新宿GR学園                                     |
+| deep-okubo    | Bellemule（深大久保DJ＆ダンスアカデミー）        |
+| deep-okubo-k  | 輝きノスタルジア（深大久保DJ＆ダンスアカデミー） |
+| shinsaibashi  | OKINI☆PARTY'S（心斎橋演芸高校）                  |
+| ikebukuro     | 池袋電音部（池袋空乗院高校）                     |
+| neotokyo      | 東京電脳（東京電脳学園）                         |
+| neonakano     | 中野電脳（中野電脳学園）                         |
+| shimokitazawa | Ma'Scar'Piece（北沢音箱高校）                    |
+
+**Category**
+Working category IDs include \`news\` (the default), \`event\`, \`goods\`, \`comic\`, \`movie\`, \`music\` or \`livearchives\`.
+
+`,
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    radar: [
+        {
+            source: ['denonbu.jp/news'],
+            target: '/news',
+        },
+        {
+            source: ['denonbu.jp/event'],
+            target: '/news/event',
+        },
+        {
+            source: ['denonbu.jp/goods'],
+            target: '/news/goods',
+        },
+        {
+            source: ['denonbu.jp/comic'],
+            target: '/news/comic',
+        },
+        {
+            source: ['denonbu.jp/movie'],
+            target: '/news/movie',
+        },
+        {
+            source: ['denonbu.jp/music'],
+            target: '/news/music',
+        },
+        {
+            source: ['denonbu.jp/livearchives'],
+            target: '/news/livearchives',
+        },
+
+        {
+            source: ['denonbu.jp/area/:area'],
+            target: '/news/:area',
+        },
+    ],
+    name: '新闻',
+    maintainers: ['outloudvi'],
+    handler,
+};
+
+const AREA_NAMES = {
+    akiba: '外神田文芸高校',
+    harajuku: '神宮前参道學園',
+    azabu: '港白金女学院',
+    shibuya: '帝音国際学院',
+    kabuki: '真新宿GR学園',
+    'deep-okubo': 'Bellemule（深大久保DJ＆ダンスアカデミー）',
+    'deep-okubo-k': '輝きノスタルジア（深大久保DJ＆ダンスアカデミー）',
+    shinsaibashi: "OKINI☆PARTY'S（心斎橋演芸高校）",
+    ikebukuro: '池袋電音部（池袋空乗院高校）',
+    neotokyo: '東京電脳（東京電脳学園）',
+    neonakano: '中野電脳（中野電脳学園）',
+    shimokitazawa: "Ma'Scar'Piece（北沢音箱高校）",
+
+    news: '新闻',
+    event: '活动',
+    goods: '商品',
+    comic: '漫画',
+    movie: '影片',
+    music: '音乐',
+    livearchives: 'Live留档',
+};
+
+const BASE_URL = 'https://denonbu.jp/backend-api/v1.0.0/';
+const COMMON_HEADERS = {
+    'X-API-KEY': 'FVpHcMLqyf7v2EubqiLxznC9gVMqBDFFMt4zvkS2',
+};
+const PRIMARY_CATEGORIES = new Set(['news', 'event', 'goods', 'comic', 'movie', 'music', 'livearchives']);
+const CACHE_TOKEN_KEY = 'denonbu-news';
+
+async function getToken(): Promise<string> {
+    const cacheToken = await cache.get(CACHE_TOKEN_KEY);
```

Comment:
```suggestion
    const cacheToken = await cache.get(CACHE_TOKEN_KEY, false);
```
docs: https://docs.rsshub.app/joinus/advanced/use-cache#cache-get-key-refresh